webpack/assets/javascripts/react_app/components/HostDetails/index.js

```diff
@@ -79,6 +79,11 @@ const HostDetails = ({
       .split('?')[0] // Remove query params
   );
 
+  const filteredTabs =
+    tabs?.filter(
+      tab => !slotMetadata?.[tab]?.hideTab?.({ hostDetails: response })
```

Comment:
I like this solution, can you also add the same condition in https://github.com/theforeman/foreman/blob/develop/webpack/assets/javascripts/react_app/components/HostDetails/Tabs/TabRouter/index.js#L22 ?

I think it could help us with avoiding the condition in each component tab, e.g:
https://github.com/Katello/katello/pull/9942/files#diff-ace943f6df83ff859b40ba000579849963c04319b84be2d089a26587077f18a3R65

Reply:
hmm, but where do I get the `slotMetadata`? :thinking:

Reply:
> I think it could help us with avoiding the condition in each component tab, e.g:
https://github.com/Katello/katello/pull/9942/files#diff-ace943f6df83ff859b40ba000579849963c04319b84be2d089a26587077f18a3R65

But that code is for an overview card, not a tab.. :question:

Reply:
Oh I see, I thought it was needed also for the tab component for some reason when looking at the code, but I guess after it's filtered there's no chance that it'd be rendered